doc/stages/filters.icp.rst

```diff
@@ -78,6 +76,17 @@ The metadata output might start something like:
 Options
 --------
 
-None.
+max_iter
+  Maximum number of iterations. [Default: **100**]
+
+max_similar
+  Max number of similar transforms to consider converged. [Default: **0**]
+
+mse_abs
+  Absolute threshold for MSE. [Default: **1e-12**]
```

Comment:
In both the documentation and in the usage text in the code, might be helpful to use the full phrase instead of the acronym.